File: VB/WinSample.Win/WinApplication.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp.Win

Namespace WinSample.Win
	Partial Public Class WinSampleWindowsFormsApplication
		Inherits WinApplication
		Public Sub New()
			InitializeComponent()
		End Sub
		Private Sub WinSampleWindowsFormsApplication_DatabaseVersionMismatch(ByVal sender As Object, ByVal e As DevExpress.ExpressApp.DatabaseVersionMismatchEventArgs) Handles MyBase.DatabaseVersionMismatch
			e.Updater.Update()
			e.Handled = True
		End Sub
	End Class
End Namespace

File: CS/WinSample.Win/WinApplication.cs
using System;
using DevExpress.ExpressApp.Win;

namespace WinSample.Win {
    public partial class WinSampleWindowsFormsApplication : WinApplication {
        public WinSampleWindowsFormsApplication() {
            InitializeComponent();
        }
        private void WinSampleWindowsFormsApplication_DatabaseVersionMismatch(object sender, DevExpress.ExpressApp.DatabaseVersionMismatchEventArgs e) {
            e.Updater.Update();
            e.Handled = true;
        }
    }
}
